cmd/katalyst-agent/app/options/qrm/memory_plugin.go

```diff
@@ -81,6 +84,7 @@ func NewMemoryOptions() *MemoryOptions {
 		EnableMemoryAdvisor:        false,
 		EnableOOMPriority:          false,
 		EnableNonBindingShareCoresMemoryResourceCheck: true,
+		NUMABindResultResourceAllocationAnnotationKey: consts.QRMResourceAnnotationKeyNUMABindResult,
```

Comment:
The variable `EnableNUMAAllocationReactor` is also explicitly set to `false`,Similarly, all newly added flags should explicitly be assigned default values.